fixes.sh

```diff
@@ -19,5 +19,18 @@ fi
 echo "Removing ~/.npmrc to fix RHIDP-4410"
 rm -rf ~/.npmrc
 
+# handle dynamic-plugins config override
+DYNAMIC_PLUGINS_DEFAULT="/opt/app-root/src/configs/dynamic-plugins/dynamic-plugins.yaml"
+DYNAMIC_PLUGINS_OVERRIDE="/opt/app-root/src/configs/dynamic-plugins/dynamic-plugins.override.yaml"
+LINK_TARGET="/opt/app-root/src/dynamic-plugins.yaml"
+
+if [ -f "$DYNAMIC_PLUGINS_OVERRIDE" ]; then
+    echo "Using dynamic-plugins.override.yaml"
+    ln -sf "$DYNAMIC_PLUGINS_OVERRIDE" "$LINK_TARGET"
+else
```

Comment:
```suggestion
elif [ -f "/opt/app-root/src/configs/dynamic-plugins.yaml" ]; then
    echo "[warn] Using legacy dynamic-plugins.yaml. This method is deprecated. You can override the dynamic plugins configuration by renaming your file into configs/dynamic-plugins/dynamic-plugins.override.yaml"
    ln -sf "/opt/app-root/src/configs/dynamic-plugins.yaml" "$LINK_TARGET"
else
```

I think this can be updated in a backward compatible way, so it doesn't break existing users, and doesn't create any Git conflicts.